src/Blacklist.php

```diff
@@ -528,8 +528,8 @@ public function processBlackList(&$value)
             $value->serial = $value->mserial;
         }
 
-        if (property_exists($value, 'ipaddress') || property_exists($value, 'ip')) {
-            $property = property_exists($value, 'ipaddress') ? 'ipaddress' : 'ip';
+        if (property_exists($value, 'ipaddress') || property_exists($value, 'ip') || property_exists($value, 'ips')) {
+            $property = (property_exists($value, 'ipaddress') ? 'ipaddress' : property_exists($value, 'ips')) ? 'ips' : 'ip';
```

Comment:
Maybe it would be more appropriate to do a loop.
```suggestion
        foreach (['ip', 'ips', 'ipaddress'] as $property) {
```